packages/opentelemetry-plugin-postgres/package.json

```diff
@@ -46,7 +46,7 @@
     "@types/pg": "^7.11.2",
     "@types/shimmer": "^1.0.1",
     "codecov": "^3.5.0",
-    "gts": "^1.1.0",
+    "gts": "^1.0.0",
```

Comment:
Why use the earlier version of `gts`?

Reply:
AFAIK most of the places we are using `1.0.0`, (I think) this change is to make sure we are consistent everywhere. We can update the latest version (`1.1.0`) in separate PR.